src/Components/MakeVote.jsx

```diff
@@ -1,16 +1,26 @@
-import { useState,useEffect } from "react"
-import { getArticleById } from "../api"
-export const MakeVote = ({setVotes})=>{
+import { incrementVotes } from "../api"
+import { useState } from "react"
+
+export const MakeVote = (props)=>{
+    const {setVotes , article_id}= props
+    const [Error, setError]=useState(null)
```

Comment:
You shouldn't use capital case for variable names, only for classes and jsx components